e2e/steps_file.js

```diff
@@ -131,6 +136,20 @@ module.exports = function () {
       this.waitForElement(CASE_HEADER);
     },
 
+    async respondToClaim() {
+      await caseViewPage.startEvent('Respond to claim');
+
```

Comment:
blank line intentional?

Reply:
yes, it's more readable when grouped related things. start of event, then pages, and then end of event.